src/Controller/BackendController.php

```diff
@@ -210,4 +212,20 @@ public function alertsAction()
 
         return $controller->run();
     }
+
+    /**
+     * Handles the picker redirect.
+     *
+     * @param Request $request
+     *
+     * @return RedirectResponse
+     *
+     * @Route("/_contao/picker", name="contao_backend_picker")
+     */
+    public function pickerAction(Request $request)
+    {
+        $pickerBuilder = $this->container->get('contao.menu.picker_menu_builder');
+
+        return new RedirectResponse($pickerBuilder->getPickerUrl($request->query->all()));
```

Comment:
I'd prefer to pass the whole `Request $request`. Who knows, maybe somebody want's to redirect based on some headers?